### PR TITLE
refactor relations from the script and use them dinamically

### PR DESCRIPTION
**Pull request by BeAllAround**

### CODE REVIEW

File: src/pqam.js [hunk 1]
```diff
@@ -169,20 +169,31 @@ import '../node_modules/leaflet-rastercoords/rastercoords.js'
 
     
     self.load = async function(done) {
+    
+      let endpoint = (msg) => {
+        let suffix = '/api/web' + '/public/' + msg.on
+        let origin = self.config.api_endpoint // 'http://127.0.0.1:8888'
```
Comment (rjrodger) [inline src/pqam.js:175]: let's just call this `endpoint`

File: src/pqam.js [hunk 2]
```diff
@@ -169,20 +169,31 @@ import '../node_modules/leaflet-rastercoords/rastercoords.js'
...
+  
       let seneca = new Seneca({
         log: { logger: 'flat', level: 'warn' },
         plugin: {
           browser: {
-            endpoint: self.config.api_endpoint,
+            endpoint,
             headers: {
-              'X-API-Key': self.config.api_key,
+              'Authorization': 'Bearer ' + self.config.api_key,
```
Comment (rjrodger) [inline src/pqam.js:186]: call this `apikey` - more consistent with dev portal

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: src/pqam.js (hunk 2 above)
Comment (BeAllAround) [reply src/pqam.js]: Done

File: src/pqam.js (hunk 1 above)
Comment (BeAllAround) [reply src/pqam.js]: Done

**Pull request merged**